src/renderer/src/pass/deferred.rs

```diff
@@ -1,25 +1,31 @@
 use gfx;
-use gfx::traits::FactoryExt;
 use gfx::handle::Buffer;
-use gfx::Slice;
+use gfx::traits::FactoryExt;
 use cgmath::{Matrix4, SquareMatrix};
-pub use ::target::{ColorFormat, GeometryBuffer};
+
+use ConstantColorTexture;
+use pass::Pass;
+use target::{ColorFormat, GeometryBuffer};
 
 gfx_vertex_struct!(Vertex {
     pos: [i32; 2] = "a_Pos",
     tex_coord: [i32; 2] = "a_TexCoord",
 });
 
-
 pub struct Clear;
 
-impl<R> ::Pass<R> for Clear
+impl<R> Pass<R> for Clear
```

Comment:
Ah, I have updated `forward.rs` to use `pass::Pass` but not `deferred.rs`. Thanks!